wordpress-mobile/WordPress-Android
# Stats Revamp v2:  Feature Announcement

This PR adds feature announcement for Stats Revamp v2.

Fixes #16686 

NOTE:
- At the moment it is set up to show up at app launch if  local debug feature flag is enabled


TODO:
- [x] Replace the image with properly exported one
- [ ] Confirm what and when should invoke this on release


|Detail screen with date selector|
|---|
|<img width=320 src="https://user-images.githubusercontent.com/990349/171862253-3364e060-7f23-498a-bd68-fd7678d1f8ff.png" />|


To test:

Setup:
1. Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
2. Select Debug Settings
3. Find StatsRevampV2FeatureConfig under Features in development enable it and restart app
4. Launch / Re-Launch app


Test:
- Notice  the new Feature announcement dialog pops up.
- Tap on [Try it now] and ensure it opens Stats 
- Relaunch or Return Feature announcement
- Tap on [Remind me later] and it should dismiss the dialog
- Relaunch or Return Feature announcement
- Tap on X on top right hand corner and it should dismiss the dialog

To test:

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the paths

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
